### PR TITLE
ESP translation => Add note about copying Equatable props to FAQs (#1067)

### PR DESCRIPTION
**Pull request by Classy-Bear**
## Status
**READY**

## Breaking Changes
NO

## Description
Included the change from #1169

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -153,4 +153,4 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          file: ${{ matrix.folder }}/coverage/lcov.info
\ No newline at end of file
+          file: ${{ matrix.folder }}/coverage/lcov.info
```
Comment (felangel) [inline .github/workflows/main.yaml:156]: can we revert this?

File: docs/es/README.md [hunk 2]
```diff
@@ -155,6 +155,6 @@ Obtenga más información en los siguientes enlaces, que han sido aportados por
 
 - [Feature Scaffolding for VSCode](https://marketplace.visualstudio.com/items?itemName=KiritchoukC.flutter-clean-architecture) - Una extensión VSCode inspirada en los tutoriales de arquitectura limpia de [Reso Coder's](https://resocoder.com), que ayuda rápidamente a las funciones de andamio, por [Kiritchouk Clément](https://github.com/KiritchoukC).
 
-## Maintainers
+## Mantenedores
```
Comment (felangel) [inline docs/es/README.md:158]: 💯

File: docs/pt-br/README.md [hunk 3]
```diff
@@ -161,6 +161,11 @@ Learn more at the following links, which have been contributed by the community.
 
 - [Feature Scaffolding for VSCode](https://marketplace.visualstudio.com/items?itemName=KiritchoukC.flutter-clean-architecture) - A VSCode extension inspired by [Reso Coder's](https://resocoder.com) clean architecture tutorials, which helps quickly scaffold features, by [Kiritchouk Clément](https://github.com/KiritchoukC).
 
+### Written Resources
```
Comment (felangel) [inline docs/pt-br/README.md:164]: 🚀

**Comment by Classy-Bear** [comment]
Sorry for all the commits :) 
Had problems squashing the commits, 

> Thanks so much! Looks great just have the one comment about reverting the changes to the `main.yaml` workflow.

Sorry for all the commits, had some problems with pulling with my old forked repository. Give me a few minutes to revert the `main.yaml` change.

File: .github/workflows/main.yaml (hunk 1 above)
Comment (Classy-Bear) [reply .github/workflows/main.yaml]: Done! :p

Comment (felangel) [reply .github/workflows/main.yaml]: It looks like the line was deleted

File: .github/workflows/main.yaml [hunk 4]
```diff
@@ -153,4 +153,3 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
```
Comment (felangel) [inline .github/workflows/main.yaml:155]:
```suggestion
          token: ${{ secrets.CODECOV_TOKEN }}
          file: ${{ matrix.folder }}/coverage/lcov.info

```

**Pull request merged**